web_fragments/test_utils/__init__.py

```diff
@@ -0,0 +1,24 @@
+"""
+Test utilities.
+
+Since py.test discourages putting __init__.py into test directory (i.e. making tests a package)
+one cannot import from anywhere under tests folder. However, some utility classes/methods might be useful
+in multiple test modules (i.e. factoryboy factories, base test classes). So this package is the place to put them.
```

Comment:
Is this from cookie cutter? I think it makes more sense to discuss what web fragments is including here specifically.